rust/src/lib.rs

```diff
@@ -120,3 +120,4 @@ pub mod lzma;
 pub mod util;
 pub mod ffi;
 pub mod feature;
+pub mod sdp;
```

Comment:
does it make sense to have this separate from sip? Are there other protocols that we may reuse this for?

Reply:
I think yes because SDP is also used with RTP and RTSP protocols, so we could eventually reuse it.

Reply:
Ok, cool